polkadot/xcm/src/v3/traits.rs

```diff
@@ -199,6 +199,54 @@ impl TryFrom<OldError> for Error {
 	}
 }
 
+impl From<NewError> for Error {
```

Comment:
If they're the exact same, you might want to consider re-exporting the type from the new version module instead of adding in conversion functions.

Reply:
Some types appeared the same but because they had a Location or something inside, I had to make a new type. Guess I went overboard with `Error` :sweat_smile: